Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -96,16 +101,19 @@ class Digitizer
   int mMaxTimeBinsTRAP = 30;                               // Maximum number of time bins for processing adcs; should be read from the CCDB or the TRAP config
 
   // Digitization containers
-  std::vector<HitType> mHitContainer;                       // the container of hits in a given detector
-  std::vector<MCLabel> mMergedLabels;                       // temporary label container
-  std::array<SignalContainer, kNdet> mSignalsMapCollection; // container for caching signals over a timeframe
-  std::array<DigitContainer, kNdet> mDigitsCollection;      // container for caching digits for paralellization
+  std::vector<HitType> mHitContainer;                            // the container of hits in a given detector
+  std::vector<MCLabel> mMergedLabels;                            // temporary label container
+  std::array<SignalContainer, kNdet> mSignalsMapCollection;      // container for caching signals over a timeframe
+  std::array<DigitContainer, kNdet> mDigitsCollection;           // container for caching digits for paralellization
+  std::deque<std::array<SignalContainer, kNdet>> mPileupSignals; // container for piled up signals
```

Comment:
looks like the include for this is missing

Reply:
I couldn't have caught this on my machine (macOS), for some reason it is not necessary to actually include deque to make it compile. Of course, I will fix this.